lib/OrdinaryDiffEqBDF/src/algorithms.jl

```diff
@@ -1,11 +1,61 @@
-"""
-E. Alberdi Celayaa, J. J. Anza Aguirrezabalab, P. Chatzipantelidisc. Implementation of
-an Adaptive BDF2 Formula and Comparison with The MATLAB Ode15s. Procedia Computer Science,
-29, pp 1014-1026, 2014. doi: https://doi.org/10.1016/j.procs.2014.05.091
+function BDF_docstring(description::String,
```

Comment:
BDF fill in

Description for:
SBDF

Reference for:
MEBDF2

Both for:
DImplicitEuler
DABDF2
DFBDF

Reply:
DImplicitEuler is Implicit Euler for implicit DAE form. It uses an apriori error estimator for adaptivity based on a finite differencing approximation from SPICE.

DABDF2 is a fully implicit implementation of BDF2 which uses the adaptivity scheme from https://www.sciencedirect.com/science/article/pii/S1877050914002683

DFBDF is a fully implicit implementation of FBDF based on Shampine's https://www.degruyter.com/document/doi/10.1515/JNMA.2002.291/html?lang=en

Reply:
MEBDF2 is https://www.sciencedirect.com/science/article/pii/S0377042700004635

Reply:
SBDF methods are implicit-explicit (IMEX) methods designed for SplitODEFunction equations which reduce the size of the implicit handling to a subset of the equations. It's similar to the additive Runge-Kutta methods in splitting mode, like KenCarp4, but instead using a multistep BDF approach. https://epubs.siam.org/doi/10.1137/0732037